### ISSUE TITLE
Python 3.8 Zip lambda failing in the first invocation, but succeeds second time onwards

### ISSUE THREAD

**Issue by mummoo** [localstack/localstack#3934]
# Type of request: This is a ...

[ X] bug report
[ ] feature request

# Detailed description
I deploy a Python 3.8 Lambda using ZIP package option. The file is ~30MB.

I do a listing of the functions using "awslocal lambda list-functions". This shows that the function is created.

I then invoke the function using a SQS message, and see this failure (when DEBUG=1 is set):

File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 231, in run_lambda_executor
          |     raise InvocationException('Lambda process returned error status code: %s. Result: %s. Output:\n%s' %
          | localstack.services.awslambda.lambda_executors.InvocationException: Lambda process returned error status code: 1. Result: {"errorType":"Runtime.ImportModuleError","errorMessage":"Unable to import module 'mymodule': No module named 'mymodule'"}. 

Lambda handler is in the file "mymodule.py" as "lambda_handler".

However, the second invocation through SQS, the lambda gets run without any issues.

It seems to randomly work as well. What is the reason for the first time failure even the lambda function is created properly?

...

## Expected behavior
Lambda invocation works without any errors in the first invocation.
...

## Actual behavior
Lambda invocation fails with Runtime.ImportModuleError  in the first invocation.
...

# Steps to reproduce

Upload a ZIP lambda (python 3.8)
Invoke it
Observe random failures in the verify invocation.

## Command used to start LocalStack
    image: localstack/localstack:latest in docker-compose.
...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
awslocal for create function, and using SQS for triggering the lambda

...

**Comment by mgagliardo** [comment]
Hello @mummoo could you please share the docker-compose, script and exact steps you are using? I can't seem to reproduce this with [the following setup](https://github.com/localstack/localstack/issues/3827#issuecomment-813090864). Thanks!

**Comment by mummoo** [comment]
Thanks @mgagliardo for taking a look.Let me try to create a test set-up and share here.

Is there anyway to have a detailed debug with localstack (beyond setting DEBUG=1)?

**Comment by mgagliardo** [comment]
Hello @mummoo, `DEBUG=1` is the most detailed you could always get.

Maybe it would also help to put some parts of the code you are using and/or docker compose as well as output logs and commands so I can reproduce. Cheers!

**Comment by whummer** [comment]
> Is there anyway to have a detailed debug with localstack (beyond setting DEBUG=1)?

@mummoo you could try enabling `LS_LOG=trace` which will print a detailed trace of all request/response messages. Please be careful with large messages (e.g., content of Lambda zip files) getting printed on the console, as they may produce large amounts of log output. Hope that helps, thanks.

**Comment by mummoo** [comment]
@mgagliardo  I enabled LS_LOG=1, and I got one message which I think may be useful:

> REPORT RequestId: 9965e212-c3f1-11ac-616b-fe3be783ad4a	Init Duration: 142.19 ms	Duration: 3.73 ms	Billed Duration: 4 ms	Memory Size: 1536 MB	Max Memory Used: 24 MB
2021-04-28T09:41:45:WARNING:bootstrap.py: Thread run method <function LambdaExecutor.execute.<locals>.do_execute at 0x7fbdd3e47d30>(None) failed: Lambda process returned error status code: 1. Result: {"errorType":"Runtime.ImportModuleError","errorMessage":"Unable to import module 'X'.: No module named 'X'"}. 

Any particular reason same would succeed without any change? any race condition related to size of the lambda zip file? (mine is close to 30MB).

PS: I am still trying to a do a repro set-up. will share once I have it.
